docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/payone/integration-in-the-back-office/payone-integration-in-the-back-office.md

```diff
@@ -1,26 +1,33 @@
 ---
-title: "Payone: Integration in the Back Office"
-description: With Payone, your customers can pay with common payment methods, such as credit card and PayPal.
+title: "Payone ACP App"
+description: With Payone, your customers can pay with common payment methods, such as credit card, PayPal, Prepayment and Klarna.
 template: howto-guide-template
-last_updated: Aug 01, 2024
+last_updated: Aug 30, 2024
 redirect_from:
    - /docs/aop/user/apps/payone.html
    - /docs/acp/user/apps/payone.html
    - /docs/pbc/all/payment-service-providers/payone/payone.html
    - /docs/pbc/all/payment-service-provider/202311.0/third-party-integrations/payone/integration-in-the-back-office/payone-integration-in-the-back-office.html
 ---
 
-The [Payone](https://www.payone.com/DE-en?ref=spryker-documentation) app lets your customers make payments with common payment methods, such as credit card and PayPal.
+The [Payone](https://www.payone.com/DE-en?ref=spryker-documentation) app lets your customers make payments with common payment methods, such as credit card, PayPal, Prepayment and Klarna.
 
-{% info_block infoBox "Info" %}
+The Payone integration in Spryker is part of the App Composition Platform and supports both the default Storefront Yves and Spryker GLUE APIs.
 
-We support only credit card and PayPal payment methods for Payone. If you want to use either or both of these payment methods, you must have them in your Payone contract. However, even if your contract with Payone includes other payment methods, you can not use them, as we do not support them yet.
+You can have multiple accounts with Payone. For example, you can have different Payone credentials per store, which we support as well.
 
-{% endinfo_block %}
+## Supported business models & Payment methods
+The Payone App supports only the B2B and B2C business models.
 
-You can have multiple accounts with Payone. For example, you can have different Payone credentials per store, which we support as well.
+The following payment methods are currently supported by the Payone App:
```

Comment:
Are you referring to Payone or Payone app? Meaning that Payone can support more payment methods

Reply:
Payone App. The list of payment methods is what we now support